charts/novelty/templates/_metrics.tpl

```diff
@@ -0,0 +1,36 @@
+{{/*
+Prometheus Metrics Reporter Configuration
+*/}}
+{{- define "novelty.metricsJdkOptions" -}}
+{{- if .Values.metrics.csv.enabled }}
+-Dthatdot.novelty.metrics-reporters.0.type=csv
+-Dthatdot.novelty.metrics-reporters.0.period={{ .Values.metrics.csv.period }}
+-Dthatdot.novelty.metrics-reporters.0.log-directory={{ .Values.metrics.csv.logDirectory }}
+{{- end }}
+{{- if .Values.metrics.influx.enabled }}
+-Dthatdot.novelty.metrics-reporters.1.type=influxdb
+-Dthatdot.novelty.metrics-reporters.1.database={{ .Values.metrics.influx.database }}
+-Dthatdot.novelty.metrics-reporters.1.period={{ .Values.metrics.influx.period }}
+-Dthatdot.novelty.metrics-reporters.1.scheme={{ .Values.metrics.influx.scheme }}
+-Dthatdot.novelty.metrics-reporters.1.host={{ .Values.metrics.influx.host }}
+-Dthatdot.novelty.metrics-reporters.1.port={{ .Values.metrics.influx.port }}
```

Comment:
Question: if influx is enabled, but csv is disabled. Is this configuration still valid? Like there is a `metrics-reporters.1`, but no `metrics-reporters.0`?

Reply:
I guess there's a similar questions as well for the below JMX metrics.

Reply:
These are great thoughts. I just tested it out and disabling CSV does still generate working configuration for Infllux and JMX

Disabling CSV and Influx, JMX also still works